locale/tdt.js

```diff
@@ -0,0 +1,66 @@
+//! moment.js locale configuration
+//! locale : Tetun Dili (East Timor) [tdt]
```

Comment:
you need to add an author line here (check the other locales for example). A good place to credit @marobo or anybody else who helped.